go.mod

```diff
@@ -21,11 +21,14 @@ require (
 	golang.org/x/mod v0.16.0
 	golang.org/x/net v0.22.0
 	google.golang.org/grpc v1.62.1
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Is this the right YAML package? I know we use another package in other places.

Reply:
you are right, we tend to use `sigs.k8s.io/yaml v1.3.0 // indirect`